go-selinux/selinux_linux.go

```diff
@@ -0,0 +1,721 @@
+// +build linux
```

Comment:
If the idea is to keep the dummy code in this repo, I think we need a more specific tag here as well.  Linux users may have a kernel that doesn't have SELinux compiled in, or they may prefer the stubs for a particular build for some other reason.

Reply:
SELinux code will just work without the selinux being compiled in on a linux system,  This is not an issue on non SELinux machines.

Reply:
> SELinux code will just work without the selinux being compiled in on a linux system, This is not an issue on non SELinux machines.

I've filed #32 with a stub tag.  Can we continue this discussion there?